src/funding/paylater/config.jsx

```diff
@@ -32,23 +20,42 @@ export function getPaylaterConfig() : FundingSourceConfig {
             BUTTON_LAYOUT.VERTICAL
         ],
 
-        Label: flexLabel,
+        Label: ({ logo }) => logo,
+
+        Logo: ({ logoColor, nonce, fundingEligibility }) => {
+            const paylaterEligibility = fundingEligibility.paylater;
 
-        Logo: flexLabel,
+            let text;
+
+            if (paylaterEligibility && paylaterEligibility.products && paylaterEligibility.products.payIn4 && paylaterEligibility.products.payIn4.eligible) {
```

Comment:
Would it be better to check against `paylaterEligibility.products.flex` in case there's a scenario where `paylaterEligibility.products.payIn4.eligible === false`, but `paylaterEligibility.products.credit.eligible === true` since the Pay Later button will be encompassing both?

Reply:
Yep, very fair, I'll check for all three options.

Reply:
Fixed